pkg/bgp/speaker/speaker.go

```diff
@@ -127,27 +143,37 @@ func (s *Speaker) OnDeleteService(svc *slim_corev1.Service) {
 		svc: nil,
 		eps: nil,
 	})
+	return nil
 }
 
 // OnUpdateEndpoints notifies the Speaker of an update to the backends of a
 // service.
-func (s *Speaker) OnUpdateEndpoints(eps *slim_corev1.Endpoints) {
+func (s *MetalLBSpeaker) OnUpdateEndpoints(eps *slim_corev1.Endpoints) error {
+	if s.shutDown() {
+		return ErrShutDown
+	}
+
```

Comment:
Nit: Just from a consistency, some funcs have this extra line here and some don't

Reply:
this should now be consistent.